packages/sdks/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -380,10 +430,10 @@ class DescopeWc extends BaseDescopeWc {
       const subscribeId = this.stepState.subscribe(() => {
         // after state was updated, we want to re-subscribe the onStepChange listener
         this.stepState.unsubscribe(subscribeId);
-        this.stepState?.subscribe(this.onStepChange.bind(this));
+        this.#subscribeStepState();
       });
     }
-
+    this.stepState.forceUpdate = isCustomScreen;
```

Comment:
remind me what forceUpdate menas?

Reply:
To call the registered callbacks on every call to update (even when there is no state changes)